src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -184,12 +184,25 @@ public function testAddResourceInvalidLocales($locale)
      */
     public function testAddResourceValidLocales($locale)
     {
+        if (null === $locale) {
+            $this->markTestSkipped('null is not a valid locale');
```

Comment:
why not remove `null` from getValidLocalesTests instead? this would look legit to me.

Reply:
because of https://github.com/symfony/symfony/pull/32415#discussion_r300871348 I don't know it seems that it used in places that can be null